contents/blog/best-growthbook-alternatives.mdx

```diff
@@ -0,0 +1,424 @@
+
+### How does LaunchDarkly compare to GrowthBook?
+
+GrowthBook positions itself as an open source alternative to LaunchDarkly, so it is no surprise their functionality is similar. Both focus specifically on feature flags and A/B testing for enterprises, but GrowthBook is missing some of the advanced features of LaunchDarkly.
+
+<ComparisonTable column1="LaunchDarkly" column2="GrowthBook">
+  <ComparisonRow column1={false} column2={true} feature="Self-serve" description="Free to try, no mandatory sales calls" />
+  <ComparisonRow column1={false} column2={true} feature="Open source" description="Audit code, contribute to roadmap, and build integrations" />
+  <ComparisonRow column1={true} column2={true} feature="Feature flags" description="Deploy features safely with targeting and percentage rollouts" />
+  <ComparisonRow column1={true} column2={false} feature="Local evaluation" description="Use local, cached flag values to increase speed" />
+  <ComparisonRow column1={true} column2={true} feature="Automations" description="Trigger flag states based on schedules, triggers" />
+  <ComparisonRow column1={true} column2={true} feature="A/B testing" description="Run multivariate tests and see the impact of changes with custom goals and reports" />
+  <ComparisonRow column1={true} column2={false} feature="Funnel tests" description="Use conversion funnels as goals for your A/B test" />
+  <ComparisonRow column1={false} column2={false} feature="Advanced analytics" description="Custom SQL, paths, and retention analysis with flag data and beyond." />
+  <ComparisonRow column1={true} column2={"Edit only"} feature="API" description="Evaluate and edit flags and A/B tests with an API" />
+  <ComparisonRow column1={true} column2={true} feature="Governance" description="Audit log, roles-based access, permissions" />
+  <ComparisonRow column1={"First party"} column2={"Third party"} feature="Data source" description="Source of data used by flags and A/B tests" />
+</ComparisonTable>
+
+### Why do companies use LaunchDarkly?
+
+According to [G2 reviews](https://www.g2.com/products/launchdarkly/reviews#reviews), users appreciate these aspects of LaunchDarkly:
+
+1. **SDKs:** Users appreciate how easy it is to integrate LaunchDarkly into their apps thanks to the range of SDKs they provide like JavaScript, Python, Android, and iOS.
+
+2. **Automations:** LaunchDarkly provides automations like scheduled rollouts, rollout templates, DevOps pipeline integrations, and stale flag cleanups. Developers mention this as A big selling point.
+
+3. **Speed and availability:** High uptime and speed are critical for developers. Users highlight local caching and edge computing integrations as critical ways LaunchDarkly supports these.
+
+> #### Bottom line
+> LaunchDarkly has an improved feature set to GrowthBook. For enterprises, it is worth considering. For smaller teams, it is unlikely the best option because it lacks self-serve and is closed source.
```

Comment:
```suggestion
> LaunchDarkly has stronger features than GrowthBook. For enterprises, it is worth considering. For smaller teams, it is unlikely the best option because it lacks self-serve and is closed source.
```